README.md

```diff
@@ -1,4 +1,4 @@
-[![Codacy Badge](https://api.codacy.com/project/badge/Grade/bc446ce0a9bd4f81b3258c50f95e01b5)](https://app.codacy.com/app/dockstore/dockstore?utm_source=github.com&utm_medium=referral&utm_content=dockstore/dockstore&utm_campaign=Badge_Grade_Dashboard)
+[![Codacy Badge](https://app.codacy.com/project/badge/Grade/c583635906d84de5a1b5f62068fc26be)](https://www.codacy.com/gh/dockstore/dockstore-cli/dashboard?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=dockstore/dockstore-cli&amp;utm_campaign=Badge_Grade)
```

Comment:
The ampersands are escaped here, but raw in another link a few lines down.  Out of curiosity, was there a reason, is one way more correct than the other, etc?